### DISCUSSION TITLE
NDJSON support

### DISCUSSION THREAD

**Discussion by franchb** [ohler55/ojg#79] [Q&A] [answered]
Hi, Peter!

Thank you for so fast JSON parser with a pretty API. Just one question -- are there any guidelines on parsing NDJSON stream using this library?

**Comment by ohler55** [comment, marked as answer]
OjG handles NDJSON Try the cmd/oj on a file with multiple JSONs in it. Any white space separators work. Not just `\n` that NDJSON requires.

**Reply by franchb** [reply]
Thank you!
